WcaOnRails/spec/models/team_spec.rb

```diff
@@ -0,0 +1,75 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Team do
```

Comment:
Note for the entire test file: Using the `@variable` notation in test DSL is completely inconsistent with the rest of our codebase. Use `let(:variable) { block_to_initialize }` and `let!(:variable) { ... }` instead, as per the rest of our `specs` folder. Pick any random test to see examples of how this is done. (Note: `let` is only executed if you actually use the variable in a test scenario, whereas `let!` is _always_ executed)

This may also explain your test failure: `@variable` is a Ruby instance variable that is bound to god knows which instance in this test scenario because specs are set up using `rspec` which does a lot of reflection shenanigans under the hood.

Reply:
Okay got it, I've changed to let variables.